File: Array 101/N_and_its_Double_exits.cpp
class Solution {
public:
    bool checkIfExist(vector<int>& arr) {
        int flag = 0;
        for(int i=0;i<arr.size();i++)
        {
            for(int j=0;j<arr.size();j++)
            {
                if(arr[j]==2*arr[i] && i!=j)
                {
                    flag++;
                    break;
                }
            }
        }
        if(flag>0)
            return true;
        else
            return false;
    }
};

File: Array 101/N_and_its_Double_exits.py
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        counter = {} 
        for i in range(len(arr)):
            if arr[i] in counter:
                return True
            counter[arr[i]*2]=1 
            if arr[i]%2==0:
                  counter[arr[i]//2]=1
        return False